lib/parse.js

```diff
@@ -13,12 +13,15 @@ var defaults = {
     depth: 5,
     parameterLimit: 1000,
     plainObjects: false,
+    prefix: '',
     strictNullHandling: false
 };
 
 var parseValues = function parseQueryStringValues(str, options) {
     var obj = {};
-    var parts = str.split(options.delimiter, options.parameterLimit === Infinity ? undefined : options.parameterLimit);
+    var cleanStr = str.indexOf(options.prefix) === 0 ? str.substr(options.prefix.length) : str;
```

Comment:
please never use `substr`; it's either nonstandard or deprecated. Always use `slice`.